e2e/mock-api/tsconfig.spec.json

```diff
@@ -5,5 +5,5 @@
     "module": "commonjs",
     "types": ["jest", "node"]
   },
-  "include": ["**/*.test.ts", "**/*.spec.ts", "**/*.d.ts"]
+  "include": ["**/*.test.ts", "**/*.spec.ts", "**/*.d.ts", "jest.config.ts"]
```

Comment:
Why do all the files within each app's configuration have a `.ts` extension when the app itself doesn't use TypeScript?

Reply:
mostly because of the default nx configuration. the mock api has no tests and we don't actually use the config. But we can add tests and change this to include js files if we were to use the config.

so long story short - default code generation

Reply:
Okay, got it!